pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -150,6 +151,30 @@ On publish, Avo will merge your Avo tracking plan with any existing events in Le
 
 Avo will not impact any events that are only defined in Lexicon, or events that have not been added to the Lexicon integration in Avo.
 
+<img
+  src={require('../../../images/publishing_rudderstack.svg')}
+  alt="Amplitude"
+  style={{ margin: '32px 0 -32px 0' }}
+/>
+
+## <a name="rudderstack"></a> RudderStack
+
+The integration to [RudderStack](https://rudderstack.com/blog/dogfooding-at-rudderstack-tracking-plans-part-1/) allows for your Tracking Plan to be published into RudderStack whenever branches are merged in your workspace or with a push of a button.
```

Comment:
Hemingway adjustments:

The Rudderstack integration enables you to publish your Avo Tracking Plan into RudderStack. You can configure the integration to publish whenever a branch is merged or with a push of a button.